### PR TITLE
Refactor field props

### PR DESCRIPTION
**Pull request by KaiVolland**
## Description

This refactors the props of the editor fields.

Most of the fields already had props like this: 
```
{
  value: string;
  onChange: (newValue: string) => void;
}
```
but some of the fields used to name the value field according to the type of the field like "radius", "color", ... 

This PR unifies this diffrences so all fields now have `value` as prop.
This PR also updates and unifies the declaration of default props for components to enhance the overview and readability.

:warning: As this updates a lot of component props this is considered as a breaking change.

## Do you introduce a breaking change?

- [x] Yes
- [ ] No
- [ ] I am unsure (no worries, we'll find out)

### CODE REVIEW

File: src/Component/NameField/NameField.tsx [hunk 1]
```diff
@@ -33,13 +33,9 @@ import {
 
 import './NameField.less';
 
-// default props
-export interface NameFieldDefaultProps {
+export interface NameFieldProps {
```
Comment (jansule) [inline src/Component/NameField/NameField.tsx:36]: Could you add a migration note for this breaking change to the commit message? Something along the lines of: Interface `NameFieldDefaultProps` was removed. Use `NameFieldProps` instead.

File: src/Component/RemovableItem/RemovableItem.tsx [hunk 2]
```diff
@@ -30,14 +30,9 @@ import { CloseCircleOutlined } from '@ant-design/icons';
 import React from 'react';
 import './RemovableItem.less';
 
-// default props
-export interface RemovableItemDefaultProps extends React.PropsWithChildren {
+export interface RemovableItemProps extends React.PropsWithChildren {
```
Comment (jansule) [inline src/Component/RemovableItem/RemovableItem.tsx:33]: See comment above

File: src/Component/Selectable/Selectable.tsx [hunk 3]
```diff
@@ -30,12 +30,7 @@ import React, { ReactNode, useState } from 'react';
 import './Selectable.less';
 import { SelectableItem } from './SelectableItem/SelectableItem';
 
-// default props
-export interface SelectableDefaultProps extends React.PropsWithChildren {
-}
-
-// non default props
-export interface SelectableProps extends Partial<SelectableDefaultProps> {
+export interface SelectableProps extends React.PropsWithChildren {
```
Comment (jansule) [inline src/Component/Selectable/Selectable.tsx:33]: See comment above

File: src/Component/Selectable/SelectableItem/SelectableItem.tsx [hunk 4]
```diff
@@ -30,16 +30,11 @@ import { CheckCircleOutlined } from '@ant-design/icons';
 import React from 'react';
 import './SelectableItem.less';
 
-// default props
-export interface SelectableItemDefaultProps extends React.PropsWithChildren {
+export interface SelectableItemProps extends React.PropsWithChildren {
```
Comment (jansule) [inline src/Component/Selectable/SelectableItem/SelectableItem.tsx:33]: See comment above

File: src/Component/Symbolizer/Field/BrightnessField/BrightnessField.tsx [hunk 5]
```diff
@@ -35,7 +35,7 @@ import { Expression } from 'geostyler-style';
 type InputProps = NumberExpressionInputProps['inputProps'];
 
 export interface BrightnessFieldProps extends InputProps {
-  brightness?: number;
+  value?: number;
```
Comment (jansule) [inline src/Component/Symbolizer/Field/BrightnessField/BrightnessField.tsx:38]: I would prefer the commit message to have a complete list of Components where this breaking change applies

File: src/Component/Symbolizer/Field/GraphicTypeField/GraphicTypeField.tsx [hunk 6]
```diff
@@ -35,24 +35,21 @@ import { useGeoStylerLocale } from '../../../../context/GeoStylerContext/GeoStyl
 
 const Option = Select.Option;
 
-export interface GraphicTypeFieldDefaultProps {
+export interface GraphicTypeFieldProps {
```
Comment (jansule) [inline src/Component/Symbolizer/Field/GraphicTypeField/GraphicTypeField.tsx:38]: See comment above

File: src/Component/Symbolizer/Field/LineDashField/LineDashField.tsx [hunk 7]
```diff
@@ -36,27 +36,22 @@ import { MinusOutlined, PlusOutlined } from '@ant-design/icons';
 
 import './LineDashField.less';
 
-// default props
-interface LineDashFieldDefaultProps {
-  dashArray: number[];
```
Comment (jansule) [inline src/Component/Symbolizer/Field/LineDashField/LineDashField.tsx:41]: This should get a dedicated comment in the commit message

File: src/Component/Symbolizer/GraphicEditor/GraphicEditor.tsx [hunk 8]
```diff
@@ -43,15 +43,11 @@ import { Form } from 'antd';
 import _get from 'lodash/get';
 import { getFormItemConfig } from '../../../Util/FormItemUtil';
 
-export interface GraphicEditorDefaultProps {
+export interface GraphicEditorProps {
```
Comment (jansule) [inline src/Component/Symbolizer/GraphicEditor/GraphicEditor.tsx:46]: See comment above

**Review by jansule — COMMENTED** [review]
Very nice! Just some remarks regarding the description of the breaking change.

**Comment by KaiVolland** [comment]
I'm not sure if a full list of all components is really needed but i will use squash merge and add a note there.

**Review by jansule — APPROVED** [review]
Very nice! Thanks for addressing the remarks!

**Pull request merged**